CONTRIBUTING.rst

```diff
@@ -19,15 +19,27 @@ Code Style
 * For Python we use PEP8 with E722 disabled (type too general in except clause)
   and the maximum line length set to 120 characters.
 
-* For documentation we use a maximum line length of 80 characters. For comments
-  and docstrings in Python, we prefer a line length of 80, but 120 may also be
-  used as long as consistency with surrounding code is maintained.
+* For prose (documentation, messages, comments) wrap lines at the word boundary
+  closest to or at, but not beyond, column 79. The first column is column 0.
 
-* We prefer single quoted strings except in JSON literals.
+* For code, we keep the trimmed line length under 81. A trimmed line is a line
+  in the source with leading and trailing whitespace removed. This means a line
```

Comment:
I just realized, this may be redundant since the presence of trailing whitespace is automatically a flake8 error. Unless the intent here is only to exclude the newline character.